tests/feast-wheel/pyproject.toml

```diff
@@ -0,0 +1,13 @@
+[tool.poetry]
+name = "feast-wheel"
+version = "0.1.0"
+description = ""
+authors = ["Star Dorminey <sterling.dorminey@gmail.com>"]
```

Comment:
Please use a fake email address, like nearly every other test in this repository.

Reply:
Fixed